bridge/github/iterator.go

```diff
@@ -49,8 +49,8 @@ type iterator struct {
 	// sticky error
 	err error
 
-	// number of imported issues
-	importedIssues int
+	// started
+	started bool
```

Comment:
Maybe we can use a better name for this. For now it is only used to know if the iterator has already started querying or not.

Reply:
What about dropping entirely this variable and initializing your variables in the constructor ?

Reply:
Much more better ! I didn't think about this at first sight